repos/fdbt-site/src/pages/direction.tsx

```diff
@@ -108,7 +108,8 @@ export const getServerSideProps = async (
     }
 
     const fareTypeAttribute = getRequiredSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE);
```

Comment:
If we use getFareTypeFromFromAttributes , we dont need to check to see if its defined.

Reply:
we can't use the function getFareTypeFromFromAttributes here